src/components/Juiciest/Juiciest.tsx

```diff
@@ -38,8 +37,8 @@ export const Juiciest = () => {
         side,
     } = useRecipeFilters();
     const [page, setPage] = useState(1);
-    const [allRecipes, setAllRecipes] = useState<Data[] | []>([]);
-    const { data, isLoading, isFetching } = useGetRecipesQuery({
+    const [allRecipes, setAllRecipes] = useState<RecipeData[] | []>([]);
```

Comment:
эмм, а почему не так
```suggestion
    const [allRecipes, setAllRecipes] = useState<RecipeData[]>([]);
```